src/Umbraco.Cms.Integrations.SEO.Semrush/Client/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "semrush",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "watch": "vite build --watch",
+    "lint:errors": "npm run lint -- --quiet",
+    "lint:fix": "npm run lint -- --fix",
+    "lint": "eslint src",
+    "generate:api": "openapi-ts"
+  },
+  "dependencies": {
+    "lit": "^3.2.0"
+  },
+  "devDependencies": {
+    "typescript": "^5.5.3",
+    "vite": "^5.4.1",
+    "vite-tsconfig-paths": "^5.0.1",
+    "@hey-api/openapi-ts": "^0.46.3",
+    "@typescript-eslint/eslint-plugin": "^7.13.0",
+    "@typescript-eslint/parser": "^7.13.0",
+    "@umbraco-cms/backoffice": "^14.3.0",
```

Comment:
If you use 14,3 here, we should also make sure we add that as the minimum version for the NuGet dependency too (i.e. 14.3.0 rather than 14.0.0).  That's fine to do, but just want to make sure we have these aligned.

Reply:
I'm having issue while getting content properties using UMB_CONTENT_WORKSPACE_CONTEXT, it returns undefined. Niels suggest that I should use cms version 15 at least. Should I use version 15 in this integration package?

Reply:
I just responded in Slack... but to repeat so we have it here, let's stick with what you have 
 - it's an extra server-trip but unlikely to be a noticeable delay since we are calling Semrush's API in this integration too.  And means we can align with getting all the integrations we plan to release out on 14.
 
Maybe when 15 is out we can look again and release an update that has a minimum dependency on that, and includes this optimization of avoiding a server round-trip.